### DISCUSSION TITLE
Search with no result

### DISCUSSION THREAD

**Discussion by sozaikia** [iterate-ch/cyberduck#14838] [Q&A]
Hi,

I tried to reconfigure the connection setting and successfully connected, it function as usual use, but when I try to search some thing it will stuck at the searching process. Any solution or idea for this case please?

**Comment by dkocher** [comment]
Previously discussed in #14475.

**Reply by sozaikia** [reply]
Hi,

I think this may not the same case, because i using the file name to search, example :- a filename called [123456789] and i search "56789", it stuck at searching process and return a white screen list result

**Comment by dkocher** [comment]
Let me know what protocol you are connecting with where the recursive search feature is broken.

**Reply by sozaikia** [reply]
cyberduck installer ver : 5.2.3.21496
S3.amazonaws.com:443

Is this the info you want?

**Comment by tatze96** [comment]
I can reproduce that the recursive search doesn't work correctly within the mentioned version, but I can confirm that it works as expected within the current release version.

We can't provide bug-fix updates for previous Cyberduck versions anymore. Please update to the current Cyberduck version - if possible for your currently used operating system version - to get the recursive search to work correctly again.
